ext/openssl/openssl.c

```diff
@@ -3836,6 +3836,9 @@ static EVP_PKEY * php_openssl_evp_from_zval(
 			}
 
 			if (in == NULL) {
+				if (filename) {
```

Comment:
The problem with adding a warning to `php_openssl_evp_from_zval` is that it will result in 2 warnings for some functions that use it and emit warning when the result is `NULL` (e.g. `openssl_csr_sign`, `openssl_pkcs12_export`...).

I think that it might be more useful to have a warning in `openssl_csr_new` as I can't imagine a use case where someone would pass a priv key (`Z_TYPE_P(out_pkey) != IS_NULL` is true) and would like to have it ignored if it fails (meaning silently using generated key instead). I might be of course missing something..?